cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaConnectConfigurationBuilderTest.java

```diff
@@ -514,4 +532,217 @@ public void withPluginPath() {
                 "plugin.path=/opt/kafka/plugins"
         ));
     }
+
+    @ParallelTest
+    public void testNullUserConfiguration()  {
```

Comment:
We don't need this test as user config can't be null.

Reply:
Removed